lib/rules/multiple-assertions-checker.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+const _ = require('lodash');
+const traverse = require('traverse');
+
+module.exports = {
+	meta: {
+		docs: {
+			description: 'Should contain a custom error message for multiple assertions.',
+			category: 'Possible Errors',
+			recommended: true
+		},
+		schema: []
+	},
+	create: context => {
+		return {
+			'ExpressionStatement': node => {
+
+				//for it() CallExpression's only
+				if (_.get(node, 'expression.callee.name') !== 'it') {
+					return;
+				}
+
+				const expectCallExpressions = [];
+				const expressionBodyNode = _.get(node, 'expression.arguments[1].body.body');
+				traverse(expressionBodyNode).forEach(function (x) {
+
+					//find expect() CallExpression's
+					if(_.get(x, 'callee.type') === 'Identifier' && _.get(x, 'callee.name') === 'expect') {
```

Comment:
I really like this approach, i'm not sure this will work though as traverse() is a dynamic function not just an array right?